overthebox/files/etc/hotplug.d/iface/33-conntrack

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+#
+# Clean udp conntrack when tun0 or xtun0 change state
+#
+[ "$INTERFACE" = tun0 -o "$INTERFACE" = xtun0 ] || exit 0
+
+clearConntrack() {
+	local mode
+	config_get mode $1 multipath
+	case "$mode" in
+		"off")
+			return;;
+		"master")
+			mode="on";;
+		"on");;
+		"backup");;
+		"handover");;
+		*)
+			return;;
+	esac
+
+	network_get_ipaddr if_ipaddr $1
+	/usr/sbin/conntrack -p udp -u ASSURED -q $if_ipaddr -D
```

Comment:
can you clear all udp entries, not only ASSURED ?
some entries in a non-standard status should be not cleared